lakefile.lean

```diff
@@ -10,10 +10,10 @@ open Lake DSL
 require "leanprover-community" / "batteries" @ git "nightly-testing"
 require "leanprover-community" / "Qq" @ git "master"
 require "leanprover-community" / "aesop" @ git "master"
-require "leanprover-community" / "proofwidgets" @ git "main"
+require "leanprover-community" / "proofwidgets" @ git "v0.0.51"
```

Comment:
Why do we need this? And if so, why not `52-pre`?

Reply:
🏎️